src/coreclr/jit/lower.cpp

```diff
@@ -96,6 +119,40 @@ bool Lowering::IsSafeToContainMem(GenTree* parentNode, GenTree* childNode)
     return true;
 }
 
+//------------------------------------------------------------------------
+// IsSafeToContainMem: Checks for conflicts between childNode and grandParentNode
+// and returns 'true' iff memory operand childNode can be contained in ancestorNode
+//
+// Arguments:
+//    grandParentNode - any non-leaf node
+//    parentNode - parent of `childNode` and an input to `grandParentNode`
+//    childNode  - some node that is an input to `parentNode`
+//
+// Return value:
+//    true if it is safe to make childNode a contained memory operand.
+//
```

Comment:
Seems like there's a few argument names here that need to be updated, `ancestorNode` and `grandParentNode` => `grandparentNode`?

Reply:
Thanks -- will fix this subsequently so I don't retrigger all the tests.